api/main.go

```diff
@@ -2,22 +2,40 @@ package main
 
 import (
 	"database/sql/driver"
+	"strconv"
 	"time"
 
+	"os"
+
 	"github.com/gin-gonic/gin"
 	"github.com/jinzhu/gorm"
 )
 
+// IsInt checkes whether a given string is an integer
+func IsInt(s string) bool {
```

Comment:
Looks like this returns true for negative values - probably not important but worth bringing up.